### DISCUSSION TITLE
Invite to share Feedback!

### DISCUSSION THREAD

**Discussion by Krishnaj20** [Sunbird-Knowlg/Community#2] [Announcements]
Folks - the entire Knowlg building block team has been spending a significant amount of time documenting the functional and technical capabilities. The result of this ongoing exercise is our new microsite - https://knowlg.sunbird.org/learn/readme. While the microsite is still WIP, we would like the community to review our new microsite, share feedback and engage with us so that we can collaborate and make the microsite useful for all adopters of Sunbird Knowlg. 
You can log your issues [here](https://github.com/Sunbird-Knowlg/Community/discussions/categories/documentation-issues).
 
@surendrasinghs @pallakartheekreddy @krgauraw @Nadiya496 @SanthoshVasabhaktula @vrayulu @ashokreddy1208 @kameshbhr @mmanimegalai @luckynvdu @reshmi-nair @sharathkashyap @gandham-santhosh @Prerna789
@chitranshu-keshav @coolbung @aayushkedia95 @ChakshuGautam @rhwarrier @alok-os @vinukumar-vs @jagadishbabu @pramodkvarma @madhucr @AmiableAnil @gouravmore @vaibhavbhuva @Jayaprakash8887 @santoshilimi @HarishGangula 
@coolbung @vijiurs @aks30 @kiranharidas187

**Comment by pramodkvarma** [comment]
Fantastic job! Huge milestone indeed! 👏👏👏

@Krishnaj20 
Few items I noticed. Under "Product & Developer Guide", all 3 sections ([content service](https://knowlg.sunbird.org/learn/product-and-developer-guide/content-service), [collection service](https://knowlg.sunbird.org/learn/product-and-developer-guide/content-service-1), and [media service](https://knowlg.sunbird.org/learn/product-and-developer-guide/content-service-2)) all have exactly same content! APIs and schemas are different, but, all initial pages under Architecture, Features, etc are exactly same!!

DIAL service should point to DIAL spec.

Btw, I also notice that left side navigation bar, under "ENGAGE" section, please add link to code repo(s). Same for [inquiry](https://inquiry.sunbird.org/learn/overview) as well.

Once again, kudos to the team!

**Reply by pramodkvarma** [reply]
I also see that few pages are still in the works.

Btw, while [Overview page](https://knowlg.sunbird.org/learn/readme) spoke of diverse use cases, all subsequent pages went into learning use case alone. I suggest that overview page be corrected to talk about "content and knowledge" organization across LMS, Commerce, IoT domains rather than actual product such as footwear etc (instead "content about footwear" is what we are talking). That way, rest of the pages talking about "content organization" can somewhat makes sense for readers. @Krishnaj20 @vrayulu
